app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/importlogins/ImportLoginsScreen.kt

```diff
@@ -167,28 +225,203 @@ private fun ImportLoginsContent(
             style = BitwardenTheme.typography.bodyMedium,
             color = BitwardenTheme.colorScheme.text.primary,
             textAlign = TextAlign.Center,
-            modifier = Modifier.standardHorizontalMargin(),
+            modifier = Modifier.standardHorizontalMargin(portrait = 12.dp),
         )
         Spacer(Modifier.height(24.dp))
         BitwardenFilledButton(
             label = stringResource(R.string.get_started),
             onClick = onGetStartedClick,
             modifier = Modifier
-                .standardHorizontalMargin()
+                .standardHorizontalMargin(portrait = 12.dp)
```

Comment:
Can we leave these as defaults. 16dp is the current default but once the app is fully updated to v3 designs, we are going to change the default to 12dp